api/kyc/api.go

```diff
@@ -217,20 +219,8 @@ func defaultIfNil(value *string, defaultValue *string) *string {
 }
 
 // Validate checks the validity of the given User
-func (u *UserRepresentation) Validate(everythingOptional bool) error {
+func (u *UserRepresentation) Validate() error {
```

Comment:
I think we should drop this method as it should not be used anymore no ?

Reply:
We still need to check some values which are not going to be stored in KC (here, attachments)